README.md

```diff
@@ -36,17 +36,17 @@ contained here, so take a moment to browse through the various files and see
 what's available:
 ```
 git clone https://github.com/kubernetes-sigs/dra-example-driver.git
-cd dra-example-driver/demo
+cd dra-example-driver
 ```
 
 From here we will build the image for the example resource driver:
 ```bash
-./build-driver.sh
+./demo/build-driver.sh
```

Comment:
This step fails in my environment:
```
#11 exporting to image
#11 exporting layers done
#11 writing image sha256:231933064c1f7e048292d362e9604294570138d20e8516a3169edc1d9d3c31b7 done
#11 naming to registry.example.com/dra-example-driver-build:golang1.22.2 done
#11 DONE 0.0s
Running 'make generate' in docker container registry.example.com/dra-example-driver-build:golang1.22.2
docker run \
	--rm \
	-e HOME=/home/ed/git/dra-example-driver \
	-e GOCACHE=/home/ed/git/dra-example-driver/.cache/go \
	-e GOPATH=/home/ed/git/dra-example-driver/.cache/gopath \
	-v /home/ed/git/dra-example-driver:/home/ed/git/dra-example-driver \
	-w /home/ed/git/dra-example-driver \
	--user $(id -u):$(id -g) \
	registry.example.com/dra-example-driver-build:golang1.22.2 \
		make generate
rm -rf /home/ed/git/dra-example-driver/deployments/helm/dra-example-driver/crds
for api in gpu/v1alpha1; do \
	rm -f /home/ed/git/dra-example-driver/api/example.com/resource/${api}/zz_generated.deepcopy.go; \
	controller-gen \
		object:headerFile=/home/ed/git/dra-example-driver/hack/boilerplate.go.txt,year=2024 \
		paths=/home/ed/git/dra-example-driver/api/example.com/resource/${api}/ \
		output:object:dir=/home/ed/git/dra-example-driver/api/example.com/resource/${api}; \
	controller-gen crd:crdVersions=v1 \
		paths=/home/ed/git/dra-example-driver/api/example.com/resource/${api}/ \
		output:crd:dir=/home/ed/git/dra-example-driver/deployments/helm/dra-example-driver/crds; \
done
Error: load packages in root "/home/ed/git/dra-example-driver/api/example.com/resource/gpu/v1alpha1": err: exit status 1: stderr: go: inconsistent vendoring in /home/ed/git/dra-example-driver:
	github.com/prometheus/client_golang@v1.16.0: is explicitly required in go.mod, but not marked as explicit in vendor/modules.txt
	github.com/urfave/cli/v2@v2.25.3: is explicitly required in go.mod, but not marked as explicit in vendor/modules.txt
	k8s.io/api@v0.30.2: is explicitly required in go.mod, but not marked as explicit in vendor/modules.txt
	k8s.io/apimachinery@v0.30.2: is explicitly required in go.mod, but not marked as explicit in vendor/modules.txt
	k8s.io/client-go@v0.30.2: is explicitly required in go.mod, but not marked as explicit in vendor/modules.txt
	k8s.io/component-base@v0.30.2: is explicitly required in go.mod, but not marked as explicit in vendor/modules.txt
	k8s.io/dynamic-resource-allocation@v0.30.2: is explicitly required in go.mod, but not marked as explicit in vendor/modules.txt
	k8s.io/klog/v2@v2.120.1: is explicitly required in go.mod, but not marked as explicit in vendor/modules.txt
	k8s.io/kubelet@v0.30.2: is explicitly required in go.mod, but not marked as explicit in vendor/modules.txt
...
```

Reply:
This was accidentally added as part of these changes. Please remove and try again:
```
commit 61dc3991299f9bb49091c8e1e02adfbc73db3117
Author: Kevin Klues <kklues@nvidia.com>
Date:   Wed Jul 10 15:32:46 2024 +0000

    Remove support for "classic" DRA

    Signed-off-by: Kevin Klues <kklues@nvidia.com>

diff --git a/Makefile b/Makefile
index e541f9b..d303a37 100644
--- a/Makefile
+++ b/Makefile
@@ -120,7 +120,7 @@ generate-clientset: generate-crds
        $(CURDIR)/pkg/$(VENDOR)/resource/clientset
    rm -rf $(CURDIR)/pkg/tmp_clientset

-generate-crds: vendor
+generate-crds:
    rm -rf $(CURDIR)/deployments/helm/$(DRIVER_NAME)/crds
    for api in $(APIS); do \
        rm -f $(CURDIR)/api/$(VENDOR)/resource/$${api}/zz_generated.deepcopy.go; \
```